src/NetMPA.Carting/NetMPA.Carting.Dal/Repositories/CartRepository.cs

```diff
@@ -0,0 +1,33 @@
+﻿using NetMPA.Carting.Dal.Interfaces;
+using NetMPA.Carting.Dal.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NetMPA.Carting.Dal.Repositories
+{
+    internal class CartRepository : ICartRepository
+    {
+        public Task<CartDao> AddAsync(CartDao cartDao)
+        {
+            throw new NotImplementedException();
```

Comment:
All services must be implemented to complete the task. This is important.

Reply:
Implemented